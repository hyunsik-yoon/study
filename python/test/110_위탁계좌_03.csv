거래일자,거래번호,거래구분,적요명,종목코드,종목명,수량,단가,거래금액,수수료,거래세,소득세,농특/주민세,세금합계,정산금액,미수변제/연체,기타대여/연체,유가잔고,예수금금잔,대용잔고,약정금액,대출(상환)금액,대출이자,대체금융회사,대체계좌번호,계좌명,받는분 표시내용,보내는분 표시내용,과표
2023-09-11,1,매수-장내,매수            ,A285130                         ,얼바인,10,100,"1,000",100,0,,,0,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-21,1,매도-장내,매도            ,A285130                         ,얼바인,4,200,800,40,80,,,80,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-21,2,매도-장내,매도            ,A285130                         ,얼바인,6,300,"1,800",60,120,,,120,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-10-12,1,매수-장내,매수            ,B2851310                        ,LA,20,200,2000,200,0,,,0,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-10-30,1,매도-장내,매도            ,B2851310                        ,LA,8,400,1600,80,160,,,160,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-10-30,2,매도-장내,매도            ,B2851310                        ,LA,12,600,3600,120,240,,,240,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
0000-01-01,,,,,이 테스트는 종목별 매도 하나에 매수가 여러개 존재하는 테스트,,,,,,,,,,,,,,,,,,,,,,,
