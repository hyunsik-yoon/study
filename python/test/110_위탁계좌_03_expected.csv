매수거래일자,매도거래일자,종목명,거래수량,총수수료,총세금합계,매수단가,매도단가,매수거래금액,매도거래금액,순수익
2023-09-11,2023-09-21,얼바인,4,80,80,100,200,400,800,240
2023-09-11,2023-09-21,얼바인,6,120,120,100,300,600,1800,960
2023-10-12,2023-10-30,LA,8,160,160,200,400,1600,3200,1280
2023-10-12,2023-10-30,LA,12,240,240,200,600,2400,7200,4320
